src/registry_schemas/schemas/admin_freeze.json

```diff
@@ -0,0 +1,31 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://bcrs.gov.bc.ca/.well_known/schemas/put_back_on",
+    "type": "object",
+    "definitions": {},
+    "title": "Admin Freeze Information Schema",
+    "required": [
+        "adminFreeze"
+    ],
+    "properties": {
+        "putBackOn": {
+            "type": "object",
+            "properties": {
+                "details": {
+                    "type": "string",
+                    "minLength": 0,
+                    "maxLength": 2000,
+                    "description": "Details which will appear in the ledger entry"
+                },
+                "action": {
+                    "type": "string",
+                    "title": "Action specifying freeze or unfreeze",
+                    "enum": [
+                        "freeze",
```

Comment:
I think a better way would be to use a flag

Reply:
changed it to flag